docs/getting_started_build_tools.md

```diff
@@ -126,9 +126,9 @@ If this is a bit complex for you, Docker might be the turn-key solution you need
 ```bash
 # You'll run this every time you want to build a keymap
 # modify the keymap and keyboard assignment to compile what you want
-# defaults are ergodox/default
+# defaults are ergodox_ez:default
 
-docker run -e keymap=gwen -e keyboard=ergodox_ez --rm -v $('pwd'):/qmk:rw edasque/qmk_firmware
+docker run -e keymap=default -e keyboard=handwired/dactyl_manuform/4x5 --rm -v $('pwd'):/qmk:rw edasque/qmk_firmware
```

Comment:
`edasque/qmk_firmware` won't work. That repo's image either needs to be rebuilt with these changes, or we need to build a new one and push it to Docker Hub.

Reply:
Just to be clear, the `edasque/qmk_firmware` here refers to a Docker Hub automated build and not the person's GitHub QMK repo. But the problem remains all the same — since the build is tied to the `edasque/qmk_firmware` GitHub repo, it will keep being out of date.

For this issue to be resolved permanently, one of the members of the [QMK organization](https://github.com/qmk) will need to make a Docker Hub automated build and tie it to the `qmk/qmk_firmware` repository directly. @jackhumbert would be the best candidate for this, as I saw he already has a DH account and a [QMK build](https://hub.docker.com/r/jackhumbert/qmk_firmware/) (but it's tied to his personal repo instead of `qmk/qmk_firmware`).

Reply:
Resolved: https://hub.docker.com/r/qmkfm/qmk_firmware/builds/